content/news/022/index.md

```diff
@@ -412,6 +412,31 @@ rusty-editor updates:
 [rg3d_twitter]: https://twitter.com/DmitryNStepanov
 [rg3d_terrain_reddit]: https://www.reddit.com/r/rust/comments/nlnfdb/timelapse_of_terrain_editing_in_rustyeditor_which/
 
+### [Nestadia][nestadia]
+
+![Zelda running on Nestadia](nestadia.png)
+
+[Nestadia][nestadia] by [@zer0x64], [@junior-n30] and [@CBenoit] is a
```

Comment:
These @ links are broken - should probably just be text.

Same on the Rustcraft section.